### ISSUE TITLE
Build favicons for pkgdown website locally

### ISSUE THREAD

**Issue by IndrajeetPatil** [easystats/easystats#407]
Currently, since we don't include favicons in the repo, they need to be generated each time the pkgdown website is built, and we would like to avoid this expensive operation.

So, what you need to do (in case you haven't done so already): 

- Create new `/pkgdown` folder in top-level directory
- Move pkgdown config file to this folder
- Run `pkgdown::build_favicons()`

For reference, see the commit in _see_ package: https://github.com/easystats/see/commit/e6dc96b0d15019dc02da1513bb65e2e7d9b306f1

## Progress tracker:

- [ ] insight 
- [ ] modelbased
- [ ] bayestestR
- [ ] effectsize
- [ ] parameters
- [ ] performance
- [ ] correlation
- [ ] report
- [ ] see
- [ ] datawizard
- [ ] easystats

**Comment by strengejacke** [comment]
We probably have to update .Rbuildignore, but maybe most/all packages already exclude the `pkgdown` folder?

**Comment by IndrajeetPatil** [comment]
Yes. I remember I had synched all of our Rbuildignore files at some point and the pkgdown folder should already be ignored.
